examples/src/bin/ech-client.rs

```diff
@@ -58,7 +58,11 @@ fn main() {
         true => None, // Force the use of the GREASE ext by skipping ECH config lookup
         false => match args.flag_ech_config {
             Some(path) => Some(read_ech(&path)),
-            None => lookup_ech_configs(&resolver, &args.arg_outer_hostname),
+            None => lookup_ech_configs(
+                &resolver,
+                &args.arg_outer_hostname,
+                args.flag_port.unwrap_or(443),
```

Comment:
Now that there's two usages of `args.flag_port` I think we should pull out a `let port = args.flag_port.unwrap_or(443)` binding on ~L49-ish so that we can use it both here and on L128 when constructing the `SocketAddr` without duplicating the `unwrap_or(...)` default.